src/test/static/javascript/dynamic-import.js

```diff
@@ -0,0 +1 @@
+import('./submodule.js').then((subModule) => console.log(subModule.subThing));
```

Comment:
Recommend a few more imports of various kinds to verify that we're early bailing in the right cases.

e.g. an import call with an identifier, an import call for a name instead of a path, etc

Reply:
Updated the dynamic-import.js to be like this:
```js
import('./submodule.js').then((submodule) => {
  console.log(submodule.subThing);
});

import('names/not/supported/yet').then((importByName) => {
  console.log('you know this is not supported yet');
});

const someString = './module.js';
import(someString).then((someModule) => {
  console.log('this is conceivably statically analyzable, but we do not do this yet');
});
```

So two cases not yet supported.

Reply:
+1